### PR TITLE
Faas invocation action for all vendors

### PR DESCRIPTION
**Pull request by jgonzalezal**
Adding documentation for our Conversation Tester, regarding the Faas invocation action errors.

### CODE REVIEW

**Comment by jgonzalezal** [comment]
@fahimmahmoodmir can you take a look

File: pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md [hunk 1]
```diff
@@ -28,6 +28,22 @@ When testing your bot and you get an error in the response, you will be able to
 ## Dialog Flow V2 Invalid Action Body
 ## Description
 This error happens when the bot has an invalid body and the Google Dialog Flow V2 is unable to parse it. It can be a wrong parameter, a value, or a missing parameter. Even though it is shown as an error in the Third Party Bots, the bot gently ignores unknown actions and the conversation will continue.
+
+## Transfer Action Error
+## Description
+For this action the required parameter is skill that has to be a string, on the other hand agentId is an optional parameter that also has to be a string value.
```
Comment (LP-fmir) [inline pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md:34]:
```
This error happens when either one or multiple of the following conditions not met
- `skill` is the required parameter and it has to be provided as a **String**
- `agentId` is an optional parameter and if provided the value must be a **String**
```

File: pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md [hunk 2]
```diff
@@ -28,6 +28,22 @@ When testing your bot and you get an error in the response, you will be able to
...
+
+## Change TTR Action Error 
+## Description
+For this action the only required value is ttrType, it has to be a string.
```
Comment (LP-fmir) [inline pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md:38]:
```
This error happens when the required parameter `ttrType` is either not provided or if provided the format of the value is not a **String**
```

File: pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md [hunk 3]
```diff
@@ -28,6 +28,22 @@ When testing your bot and you get an error in the response, you will be able to
...
+
+## Close Conversation Action Error
+## Description
+For this action there are no required parameters. In case the withoutPcs parameter is in the vendor response it has to be a boolean value.
```
Comment (LP-fmir) [inline pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md:42]: Please follow the format I provided above errors.

File: pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md [hunk 4]
```diff
@@ -28,6 +28,22 @@ When testing your bot and you get an error in the response, you will be able to
...
+## Description
+For this action the only required value is ttrType, it has to be a string.
+
+## Close Conversation Action Error
+## Description
+For this action there are no required parameters. In case the withoutPcs parameter is in the vendor response it has to be a boolean value. 
+
+## Faas Invocation Action Error
+## Description
+For this action the required values are lambdaUuid as string  and payload as object. FailOnError is an optional boolean value.
```
Comment (LP-fmir) [inline pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md:46]: Please follow the format I provided above errors.

File: pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md [hunk 5]
```diff
@@ -261,6 +293,22 @@ This error happens when not parsed messages and actions were found from the vend
 ## Description
 This error happens when the bot has an invalid action body and Lex is unable to parse it. It can be a wrong parameter, a value, or a missing parameter. Even though it is showing as an error in the Third Party Bots, the bot gently ignores unknown actions and the conversation will continue.
 
+## Transfer Action Error
```
Comment (LP-fmir) [inline pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md:296]: Please follow the format I provided above errors.

File: pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md [hunk 6]
```diff
@@ -77,6 +93,22 @@ This error happens when not parsed messages and actions were found from the vend
 ## DialogFlow CX Invalid Action Body
 ## Description
 This error happens when the bot has an invalid body and the Google Dialog Flow CX is unable to parse it. It can be a wrong parameter, a value, or a missing parameter. Even though it is shown as an error in the Third Party Bots, the bot gently ignores unknown actions and the conversation will continue.
+
```
Comment (LP-fmir) [inline pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md:96]: Please follow the format I provided above errors.

File: pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md [hunk 7]
```diff
@@ -338,6 +386,22 @@ This error happens when not parsed messages and actions were found from the vend
 ## Invalid Action Body
 ## Description
 This error happens when the bot has an invalid action body and Faas is unable to parse it. It can be a wrong parameter, a value, or a missing parameter. Even though it is showing as an error in the Third Party Bots, the bot gently ignores unknown actions and the conversation will continue.
+
```
Comment (LP-fmir) [inline pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md:389]: Please follow the format I provided above errors.

**Review by LP-fmir — CHANGES_REQUESTED** [review]
Please change the documentation as per i provided with the two examples

File: pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md (hunk 1 above)
Comment (jgonzalezal) [reply pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md:34]: Done

File: pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md (hunk 2 above)
Comment (jgonzalezal) [reply pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md:38]: done

File: pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md (hunk 4 above)
Comment (jgonzalezal) [reply pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md:46]: done

File: pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md (hunk 5 above)
Comment (jgonzalezal) [reply pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md:302]: done

File: pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md (hunk 6 above)
Comment (jgonzalezal) [reply pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md:96]: done

**Comment by jgonzalezal** [comment]
@LP-fmir I did what you suggested

File: pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md (hunk 7 above)
Comment (jgonzalezal) [reply pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md:407]: done

File: pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md [hunk 8]
```diff
@@ -28,6 +28,28 @@ When testing your bot and you get an error in the response, you will be able to
...
+
+## Close Conversation Action Error
+## Description
+For this action there are no required parameters. When the withoutPcs optional parameter is provided the format should be **Boolean**
+
+## Faas Invocation Action Error
+## Description
+This error happens when either one or multiple of the following conditions not met
+- `lambdaUuid` has not being provided as a **String**
```
Comment (LP-fmir) [inline pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md:50]: also identify which of them is optional or required

File: pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md [hunk 9]
```diff
@@ -261,6 +305,28 @@ This error happens when not parsed messages and actions were found from the vend
...
```
Comment (LP-fmir) [inline pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md:326]: describe if they are required or optional

File: pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md [hunk 10]
```diff
@@ -338,6 +404,28 @@ This error happens when not parsed messages and actions were found from the vend
...
```
Comment (LP-fmir) [inline pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md:426]: identify if they are required or optional

**Review by LP-fmir — CHANGES_REQUESTED** [review]

File: pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md (hunk 8 above)
Comment (jgonzalezal) [reply pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md]: done

File: pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md (hunk 10 above)
Comment (jgonzalezal) [reply pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md]: done

**Review by LP-fmir — APPROVED** [review]

**Pull request merged**